scripts/watcher/LongRunningExecutor.ts

```diff
@@ -51,18 +50,43 @@ export default class LongRunningExecutor implements Executor {
     });
   }
 
-  private restart(): void {
+  private async restart(): Promise<void> {
```

Comment:
Why define the return type here while it's can be calculated?

Reply:
Sure